pom.xml

```diff
@@ -105,6 +105,18 @@
       <artifactId>xml-resolver</artifactId>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <!--scope>test</scope-->
```

Comment:
Any reason why scope is commented out?  I tried to build wildfly-core with this change but got a warning about this change introducing a transitive dependency on junit:
>[WARNING] Rule 0: org.apache.maven.plugins.enforcer.BanTransitiveDependencies failed with message:
>org.wildfly.core:wildfly-core-feature-pack:pom:2.0.6.Final-SNAPSHOT
>   xerces:xercesImpl:jar:2.11.0.SP5-SNAPSHOT:compile has transitive dependencies:
>      junit:junit:jar:4.12:test
>         org.hamcrest:hamcrest-core:jar:1.3:test